checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallInferenceLogic.java

```diff
@@ -29,8 +62,8 @@
  */
```

Comment:
The above comment also needs to be updated, since you've added capabilities to this class.